docs/cluster_spec.md

```diff
@@ -159,9 +158,39 @@ Will result in the flag `--feature-gates=ExperimentalCriticalPodAnnotation=true,
 
 ### networkID
 
-On AWS, this is the id of the VPC the cluster is created in. If creating a cluster from scratch, this field doesn't need to be specified at create time; `kops` will create a `VPC` for you.
+On AWS, this is the id of the VPC the cluster is created in. If creating a cluster from scratch, this field does not need to be specified at create time; `kops` will create a `VPC` for you.
 
 ```yaml
 spec:
   networkID: vpc-abcdefg1
 ```
+
+More information about running in an existing VPC is [here](run_in_existing_vpc.md).
+
+### hooks
+
+Hooks allow the execution of a container before the installation of Kubneretes on every node in a cluster.  For intance you can install nvidia drivers for using GPUs.
+
+```
+spec:
+  # many sections removed
+  hooks:
+  - execContainer:
+      image: kopeio/nvidia-bootstrap:1.6
+```
+
+Install Ceph
+
+```
+spec:
+  # many sections removed
+  hooks:
+  - execContainer:
+      command:
+      - sh
+      - -c
+      - chroot /rootfs apt-get update && chroot /rootfs apt-get install -y ceph-common
+      image: busybox
+```
+
```

Comment:
If you're looking for something that runs continuously on all nodes in the cluster, use a DaemonSet.  If you just want to initialize something one-time on all nodes then a hook is appropriate.